### ISSUE TITLE
auto_camera fails with yaml configuration, expected DataType int32

### ISSUE THREAD

**Issue by mlohry** [Alpine-DAV/ascent#1287]
Running with an ascent_actions.yaml that includes a scene with:

```
          type: "auto_camera"
          auto_camera:
            metric: "dds_entropy"
            samples: 10
```

errors out as `samples` is being read as int64:

```
file: conduit_node.cpp
line: 14607
message:
Node::as_int32() const -- DataType int64 at path params/renders/r2/auto_camera/samples does not equal expected DataType int32
```

**Comment by cyrush** [comment]
thanks for this report, we will resolve this.

**Comment by nicolemarsaglia** [comment]
A fix was merged in with #1288 to forcibly recast the int type to avoid mismatches.
